Samples/Samples/View/CalendarEventPage.xaml.cs

```diff
@@ -104,7 +104,7 @@ async void OnEditEventButtonClicked(object sender, EventArgs e)
         {
             var modal = new CalendarEventAddPage();
 
-            var calendarName = (await Calendar.GetCalendarsAsync()).Where(x => x.Id == ViewModel.CalendarId).First().Name;
+            var calendarName = (await Calendar.GetCalendarsAsync()).First(x => x.Id == ViewModel.CalendarId).Name;
```

Comment:
Use FirstOrDefault and check for null to prevent exceptions when calendarid not found

Reply:
altered in latest commit